pycytominer/cyto_utils/cells.py

```diff
@@ -267,14 +292,20 @@ def load_compartment(self, compartment):
         return df
 
     def aggregate_compartment(
-        self, compartment, compute_subsample=False, aggregate_args=None
+        self,
+        compartment,
+        compute_subsample=False,
+        compute_counts=False,
```

Comment:
instead of adding `compute_subsample` as an argument, can you use `compute_subsample` through `aggregate_args`?

In other words, the current arguments stay the same (preferring a minimalistic core), and the function accesses the variable through `aggregate_args`? Actually no, nvm, I see now that you're also adding functionality directly here. In this case, I think how you have it makes sense. Anything passing through aggregate.py only should be included in `aggregate_args`. 

By that standard, then I think you should add `self.object_feature` to the aggregate args instead of directly to the `aggregate` call.

Something like:

```python
aggregate_args["features"] = features
aggregate_args["object_feature"] = self.object_feature
```

To me, this seems like a slightly more consistent API, but it can go either way really.

Reply:
Makes sense - https://github.com/cytomining/pycytominer/pull/129/commits/1953cffdf96cb42280ed4f3513f9048a1614ceca